services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/taydennys/TaydennysController.kt

```diff
@@ -132,6 +133,30 @@ class TaydennysController(private val taydennysService: TaydennysService) {
     fun send(@PathVariable id: UUID): HakemusResponse =
         taydennysService.sendTaydennys(id, currentUserId()).toResponse()
 
+    @DeleteMapping("/taydennykset/{id}")
+    @Operation(
+        summary = "Delete a täydennys",
+        description =
+            """
+               Deletes a täydennys.
+            """,
```

Comment:
If there's nothing else to add to the API docs, there's no reason to have a heredoc-style string here.
```suggestion
            "Deletes a täydennys.",
```

Reply:
Good catch, thanks.